types/qs/index.d.ts

```diff
@@ -54,7 +54,7 @@ declare namespace QueryString {
         interpretNumericEntities?: boolean | undefined;
     }
 
-    interface ParsedQs { [key: string]: undefined | string | string[] | ParsedQs | ParsedQs[] }
+    interface ParsedQs { [key: string]: undefined | number | string | string[] | ParsedQs | ParsedQs[]  }
```

Comment:
how can this produce a number? The only thing I can think of is using a decoder, but putting `any` here wouldn't be useful.

Reply:
Yes. Agree. I am new to this world. I was using Express it uses ParsedQS for handling request.query object. I want to pass 2 vairables limit and skip both are numbers. To pass them it needs to be number . I got confused and Raised Pr so that it is a number. What are the additional changes need to be done?

Reply:
The input for parse is a query string, and this type describes the output. Where are you passing these numbers exactly?

Reply:
Hey looks like parsing the query string is not possible. I am closing the PR. I will use zod for parsing again. Thanks.

Reply:
Parsing the query string you want is absolutely possible with qs, using a custom decoder - that just wouldn't necessitate updating the types.